### PR TITLE
Index page

### PR DESCRIPTION
**Pull request by AndrewEBolt**
This is an index page for picatic.io that will list our current examples. Built with angularjs 1.5.x and webpack

Closes picatic/picatic#259

### CODE REVIEW

File: picatic.io/app/components/example-container/example-container.template.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<figure ng-class="{'col-md-offset-2': $first}" class="grid__brick col-6 col-sm-6 col-md-4 col-xl-4 shuffle-item shuffle-item--visible" ng-repeat="example in $ctrl.examples" ng-cloak>
```
Comment (jlfama) [inline picatic.io/app/components/example-container/example-container.template.html]: .col-6 and .col-xl-4 aren't actual classes.
Even if they were, they would be redundant redundancies.

Comment (jlfama) [inline picatic.io/app/components/example-container/example-container.template.html]: `.col-*-*` classes should always be within a `.row` (or similar class) in order to get margins correct.
Ex:
```
<div class="container">
  <div class="row">
    <div class="col-xs-12 col-md-4"></div>
  </div>
</div>
```

Mostly speaking to this being a child of a `.col`, and its parent being a direct child of `.container`

File: picatic.io/app/components/header/header.module.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = function(ngModule) {
+  ngModule.component('header',  {
+    template: require('./header.template.html'),
```
Comment (jlfama) [inline picatic.io/app/components/header/header.module.js:5]: I haven't seen this `require()` stuff here before. What's the purpose/benefit?

File: picatic.io/app/services/ExamplesService.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+module.exports = function(ngModule) {
+  ngModule.factory('ExamplesService', [ function() {
+    var ExamplesService = {
+      examples: [
+        // {
```
Comment (jlfama) [inline picatic.io/app/services/ExamplesService.js]: Try not to leave "commented out" blocks in public code.
You can always look back at an older commit to see what you had there if needed.

File: picatic.io/app/components/header/header.module.js (hunk 2 above)
Comment (AndrewEBolt) [reply picatic.io/app/components/header/header.module.js:5]: This is just to let webpack know to include './header.template.html' in the bundle.js file.

**Review by jlfama — APPROVED** [review]

**Pull request merged**